src/backend/gl/src/factory.rs

```diff
@@ -148,7 +148,7 @@ impl Factory {
         else {
             let usage = match info.usage {
                 Data => gl::STATIC_DRAW,
```

Comment:
Another way could be to use `Upload` + mapping hidden behind the update commands.